### DISCUSSION TITLE
I'm a student but can't get the "@schoolar.com" because the institute only give it to their workers

### DISCUSSION THREAD

**Discussion by Test576M** [community/community#19205] [Copilot Conversations]
I'm studying at "Digital House" a software developer career and with the GitHub Copilot update I can't use it without pay it 
While reading around the forum I noticed that the "student" state comes with the benefit of using the Copilot extension for free
My problem started when I tried to register as a formal student on GitHub but the application form requires the +"@Digitalhouse.com" into mi email, soo I contacted my tutor and we spoke about it for a while, I asked for an official document that proves my attendance at DigitalHouse and the software developer career. 
I received the official document issued by the institution plus an email that told me that I cant have the +"@Digitalhouse.com" domain because I am a student, it's exclusive for teachers, tutors, and the workers from Digital House.
I can prove my student status at the institution and the career I'm doing but can't get the domain 
Can I get the student status onto GitHub despite that?
I'll attach the certificate if I need to provide any other information please ask for it and I will attach everything u need as fast as I can.
[Certificado Alumno Regular - Pablo moya.pdf](https://github.com/github-community/community/files/8960787/Certificado.Alumno.Regular.-.Pablo.moya.pdf)

**Comment by Roooklang** [comment]
I think you can, but I will recommend going to :

[Github Support](support.github.com)
[Github Community](Github.community)

Ask this question to a Github staff member and they may give you an answer. But I think you can get the students benefits!

**Reply by Test576M** [reply]
thanks u so much, i'm on my way.
